codesamples/factories.py

```diff
@@ -31,10 +31,10 @@ def initial_data():
 
             <span class=\"comment\"># Input, assignment</span>
             >>> name = input('What is your name?\\n')
-            >>> print('Hi, %s.' % name)
             <span class=\"output\">What is your name?
-            Python
-            Hi, Python.</span></code>
+            Python</span>
+            >>> print('Hi, %s.' % name)
```

Comment:
I think we should highlight f-strings while we're here!

```suggestion
            >>> print(f'Hi, {name}.')
```

Reply:
If there's no objection to this, I'm happy to commit the suggestion and merge the PR. Thank you @sblondon

Reply:
It's good for me.

There is a PR converting the string to f-string created by wkeithvan before this PR : https://github.com/python/pythondotorg/pull/1539 I guess we need to do something to it (like closing it with a link to this PR).

Reply:
I committed your suggestion and added wkeithvan and ewdurbin in the commit details.